src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -36,28 +42,54 @@ class FormIntroButtons extends React.Component {
     this.props.fetchInProgressForm(this.props.formId, this.props.migrations);
   }
 
+  toggleModal = () => {
+    this.setState({ modalOpen: !this.state.modalOpen });
+  }
+
+  startOver = () => {
+    this.toggleModal();
+    this.props.removeInProgressForm(this.props.formId, this.props.migrations);
+  }
+
   render() {
-    let resumeButton = null;
-    let firstPageButtonText = 'Continue';
-    let firstPageButtonClass = 'usa-button-primary';
     if (this.props.formSaved) {
```

Comment:
I'm going to save this for the last open SiP PR, so I don't cause a bunch of merge conflicts.